README.md

```diff
@@ -2,7 +2,7 @@
 
 # Project Name
 
-> Description the project.
+> "Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to make simple calculations and read a random math-related quote.
```

Comment:
- [ ] [OPTIONAL] Let's make sure instead of having `Project Name` on line 3 we change it to "Math Magicians" to make project description more appealing 😇

Reply:
ohhh 😄 , thank you @Amrhub. let me do it fast. 👍

Reply:
@joel-kalema Awesome! 👀